src/Datasource/Connection.php

```diff
@@ -15,7 +15,7 @@
 namespace Cake\ElasticSearch\Datasource;
 
 use Cake\ElasticSearch\Datasource\SchemaCollection;
-use Cake\Log\Log;
+use Cake\Database\Log\LoggedQuery;
```

Comment:
Yes, and have an interface for it. Right now the logger hints against the implementation which is kind of annoying.